Kd GVS bulk ingest docs (#8397)
broadinstitute/gatk: scripts/variantstore/docs/gvs-bulk-ingest-details.md

##### Running on a sample set
1. On the data tab of your workspace, select the samples that you want to include.
2. Click edit and select "Save selection as set". 
3. Name your sample set. The name you choose is what you will use as the `sample_set_name` parameter.
4. Return to the input configuration and put the sample set name into the input `sample_set_name`. We validate that the requested sample_set exists when it is input as a parameter.

#### Sample data

The [GVS beta workspace](https://app.terra.bio/#workspaces/gvs-prod/Genomic_Variant_Store_Beta) has been configured with example reblocked GVCF files that you can use to test the workflow.

The GVS workflow takes in reblocked single sample GVCF files and their corresponding index files as `input_vcfs` and `input_vcf_indexes`, respectively. 

In the Beta user workflow, the value examples are as follows:
the samples_table_name value is `sample`
the entity_id_column_name value is `sample_id`
the vcf_files_column_name value is `gvcf`
the vcf_index_files_column_name value is `gvcf_index`

> the sample_set_name value is `sample_set`

Reviewer: should this be a numbered or bullet list?

Author: Updated to a table, thanks for the suggestion to revisit the format!

# Bulk Ingest Details

To enable the ingestion of >10k samples at a time, we now support bulk ingestion. It is most straightforward to use the default instructions in [Run Your Own Samples](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/beta_docs/run-your-own-samples.md). However, if your data doesn't follow the recommended table structure, this document outlines the optional parameters you can use to use GVS.

---

#### Input descriptions

The table below describes the GVS ingest variables:

| Input variable name         | Description                                                                                                                                                                | Type |
|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------| --- |

> | samples_table_name          | (Optional) The name of the data table; This table holds the GVCFs to be ingested; `sample` is the default.                                                                 | String |

Reviewer 2: ```suggestion
| samples_table_name          | (Optional) The name of the data table; This table holds the sample IDs, the GVCFs, and their respective indexes, to be ingested; `sample` is the default.                                                                 | String |
```

| sample_id_column_name       | (Optional) User defined sample id column name; `sample_id` is the recommended name;                                                                                        | String |

> | vcf_files_column_name       | (Optional) Column that contains the GCS paths to the sample GVCF files. Determined by file inspection if not supplied.                                                     | String |

Reviewer 2: ```suggestion
| vcf_files_column_name       | (Optional) Column that contains the GCS paths to the sample GVCF files. Determined by analysis of the available column names if not supplied.                                                     | String |
```

> | vcf_index_files_column_name | (Optional) Column that contains the GCS paths to the sample GVCF index files. Determined by file inspection if not supplied.                                               | String |

Reviewer 2: ```suggestion
| vcf_index_files_column_name | (Optional) Column that contains the GCS paths to the sample GVCF index files. Determined by analysis of the available column names if not supplied.                                               | String |
```

> | sample_set_name             | (Optional) The name of the set of samples. Only required when doing subsets of data in the `sample` table.                                                                 | String |

Reviewer 2: ```suggestion
| sample_set_name             | (Optional) The name of the set of samples. Only required when doing subsets of data in the `sample` table; otherwise all the samples will be ingested.                                                                 | String |
```

| dataset_name                | Name of the BigQuery dataset used to hold input samples, filtering model data, and other tables created during the workflow.                                               | String |
| project_id                  | Name of the Google project that contains the BigQuery dataset.                                                                                                             | String |

> | call_set_identifier         | Used to name the filter model, BigQuery extract tables, and final joint VCF shards. Should begin with a letter, valid characters include A-z, 0-9, “.”, “,”, “-“, and “_”. | String |

Reviewer 2: ```suggestion
| call_set_identifier         | Used to track costs and name the filter model, BigQuery extract tables, and final joint VCF shards. Should begin with a letter, valid characters include A-z, 0-9, “.”, “,”, “-“, and “_”. | String |
```
